.github/workflows/ScenarioTest.yaml

```diff
@@ -91,104 +91,104 @@ jobs:
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=crashing_npc timeout:=5 > log.txt
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=crashing_npc osm_file:=lanelet2_map_with_private_road_and_walkway_ele_fix.osm map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=5 > log.txt
           cat log.txt | grep "Scenario Succeed"
 
       - name: Run spawn with offset scenario
         if: matrix.test_flags == 'cpp_mock_test'
         run: |
           source ~/ros2_ws/install/setup.bash
-          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=spawn_with_offset timeout:=15
+          ros2 launch cpp_mock_scenarios mock_test.launch.py scenario:=spawn_with_offset osm_file:=lanelet2_map_with_private_road_and_walkway_ele_fix.osm map_path:="$(ros2 pkg prefix kashiwanoha_map)/share/kashiwanoha_map/map" timeout:=15 > log.txt
```

Comment:
.osm file should be exactly fixed in scenario. (C++ source code)
So, please remove osm_file and map_path arguments.